services/analyzer/sequelize-column-type-getter.js

```diff
@@ -72,11 +73,14 @@ function ColumnTypeGetter(databaseConnection, schema, allowWarning = true) {
     return `ARRAY(DataTypes.${await this.perform(innerColumnInfo, innerColumnInfo.udtName, tableName)})`;
   };
 
+  // eslint-disable-next-line sonarjs/cognitive-complexity
```

Comment:
I fell like we should try to avoid introducing cognitive complexity. The complexity was already there, you added one line, so it's not your responsibility. We also should not mix refactor and feature introduction. 

**But**, I think we could prefer a refactoring task over adding `eslint-disable-next-line`. WDYT?

### Suggestion

A small refactor could be replacing all these occurrence (they are considered as "complex" by the linter, we could consider these as duplicate):

```js
case type.match(/^SMALLINT.*/i) || {}).input:
```

With:
```js
case typeStartingWith(type, 'SMALLINT'):
```

Then create a function:
```js
function typeStartingWith(type, value) {
  return type.match(new RegExp(`^${value}.*`, 'i') || {}).input;
}
```

I just checked (I did not tested it though), it's enough to reduce the complexity detected by sonarJS. It also removes 11 duplicates lines. Still, I'm aware this is not related to this actual issue. Feel free to suggest something else (however, I feel like we have to do something to address this `eslint-disable` issue).